drizztsaga/drizztsaga.tp2

```diff
@@ -359,6 +360,35 @@ ACTION_IF GAME_IS ~BGT~ BEGIN
 	BUT_ONLY
 END
 
+///////////////////////////////////////////////////////////////////////////////////////////////
+//// Optional component: Creatures, Items and XP Rebalanced                   /////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+BEGIN @1510
+DESIGNATED 4
+SUBCOMPONENT @1509
```

Comment:
I agree with @ALIENQuake, each nerfing could be a separated component inside the Weidu Group named Nerfing or something like this.